.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -10,6 +10,13 @@ it('allows installing/uninstalling an application and inviting new users', () =>
   });
   cy.reload();
 
+  // Perform a second login in case the user is logged out after installing the Sales app
+  cy.get('#login', {timeout: 0}).then($login => {
```

Comment:
Is it necessary to explicitly set a null timeout here?

Reply:
It's not necessary, it just speeds it up.